src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -1059,9 +1183,9 @@ FixedwingAttitudeControl::task_main()
 				_actuators.control[actuator_controls_s::INDEX_THROTTLE] = _manual.z;
 			}
 
-			_actuators.control[actuator_controls_s::INDEX_FLAPS] = flaps_control;
+			_actuators.control[actuator_controls_s::INDEX_FLAPS] = flaps_applied;
 			_actuators.control[5] = _manual.aux1;
-			_actuators.control[6] = _manual.aux2;
+			_actuators.control[actuator_controls_s::INDEX_AIRBRAKES] = flaperon_applied;
```

Comment:
Actually we used if exactly for that. The flaps would go down and the ailerons up which would increase the drag of the airplane without significantly changing the pitching moment of the plane.